tests/ctl/api/test_seed.py

```diff
@@ -460,7 +460,7 @@ class TestLoadSamples:
         "FIDES_DEPLOY__CONNECTORS__POSTGRES__DBNAME": "test-var-db",
         "FIDES_DEPLOY__CONNECTORS__POSTGRES__USERNAME": "test-var-user",
         "FIDES_DEPLOY__CONNECTORS__POSTGRES__PASSWORD": "test-var-password",
-        "FIDES_DEPLOY__CONNECTORS__POSTGRES__SSH_REQUIRED": "True",
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__SSH_REQUIRED": "false",
```

Comment:
Going forward, this test now reproduces the error we saw when deploying this too